jekyll-assets/_includes/head.html

```diff
@@ -1,5 +1,6 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
+<link rel="shortcut icon" href="{{ site.baseurl }}/favicon.ico">
```

Comment:
Is this different to the default behaviour of using https://www.raspberrypi.com/favicon.ico?

Reply:
Currently, local builds of the docs yell at you on the CLI about a missing favicon and lack a favicon entirely. I added this to fix the issue locally, and I _think_ it shouldn't break anything. Though your sanity check is appreciated. Hopefully that makes sense now.